README.md

```diff
@@ -1,5 +1,4 @@
-Grenade
-=======
+# Grenade
```

Comment:
Yeah, it wasn't necessary, but it made it easier to work in the documentation without having to hunt around for the correct horizontal bar character for each level.